Source/WebInspectorUI/UserInterface/Views/ConsolePrompt.js

```diff
@@ -59,9 +59,13 @@ WI.ConsolePrompt = class ConsolePrompt extends WI.View
         this._completionController = new WI.CodeMirrorCompletionController(WI.CodeMirrorCompletionController.Mode.PausedConsoleCommandLineAPI, this._codeMirror, this);
         this._completionController.addExtendedCompletionProvider("javascript", WI.javaScriptRuntimeCompletionProvider);
 
+        this.element.ariaLabel = WI.UIString("Console prompt");
+
         let textarea = this._codeMirror.getInputField();
-        if (textarea)
+        if (textarea) {
             textarea.ariaLabel = WI.UIString("Console prompt");
+            this.element.ariaLabel = null;
```

Comment:
can you assign `this.element.ariaLabel` in the else branch rather than reassigning to null?

Reply:
yeah there's no need to set it and then unset it

heck id even suggest maybe this to avoid repeating it
```js
(textArea || this.element).ariaLabel = WI.UIString("Console prompt");
```